int32[] grid_map_data
---
